### PR TITLE
1.0.8.2

### PR DESCRIPTION
**Pull request by chachasikes**
* Fix category breadcrumb
* Removed a couple duplicated function calls
* Clean up connection to external CSS
* Test mobile typography fixes & then unstage them.

### CODE REVIEW

File: includes/templates/template-single.php [hunk 1]
```diff
@@ -73,7 +71,7 @@
                         </article>
 
                     <?php endwhile; ?>
-
```
Comment (aaronhans) [inline includes/templates/template-single.php:76]: getting ready to add the per page feedback widget, I would put the custom element in here right?

File: styles/editor.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/* */
```
Comment (aaronhans) [inline styles/editor.css:1]: nice, thanks

File: includes/templates/template-single.php (hunk 1 above)
Comment (chachasikes) [reply includes/templates/template-single.php:76]: nope - do a search for PER PAGE FEEDBACK & you'll find where it goes

**Review by aaronhans — APPROVED** [review]
Excellent updates

**Pull request merged**